fem/intrules.hpp

```diff
@@ -26,9 +26,12 @@ class IntegrationPoint
 {
 public:
    double x, y, z, weight;
+   int ipID; // added by srw for Eval routine on QuadratureVectorFunctionCoefficient
```

Comment:
We should probably set a default value for ipID.

Reply:
It turns out this value was never actually set when the IntegrationRule is initially created. I just pushed a commit that at least sets this when the IntegrationRule class is initialized with a given number of points. However, I do agree an initial value should still be set when the IntegrationPoint class is created. Do you think -1 would be a good value to set initially?

Reply:
Yeah I think -1 is quite reasonable.